applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -72,6 +76,78 @@ static void print_codec(const struct bt_codec *codec)
 	}
 }
 
+static void advertising_process(struct k_work *work)
+{
+	int ret;
+	struct bt_le_adv_param adv_param;
+
+#if CONFIG_BT_BONDABLE
+	bt_addr_le_t addr;
+
+	if (!k_msgq_get(&bonds_queue, &addr, K_NO_WAIT)) {
+		char addr_buf[BT_ADDR_LE_STR_LEN];
+
+		adv_param = *BT_LE_ADV_CONN_DIR_LOW_DUTY(&addr);
+		adv_param.id = BT_ID_DEFAULT;
+
+		ret = bt_le_adv_start(&adv_param, NULL, 0, NULL, 0);
+
+		if (ret) {
+			LOG_ERR("Directed advertising failed to start");
+			return;
+		}
+
+		bt_addr_le_to_str(&addr, addr_buf, BT_ADDR_LE_STR_LEN);
+		LOG_INF("Direct advertising to %s started", addr_buf);
+	} else
+#endif
+	{
+		adv_param = *BT_LE_ADV_CONN;
+		adv_param.options |= BT_LE_ADV_OPT_ONE_TIME;
+
+		ret = bt_le_adv_start(&adv_param, ad, ARRAY_SIZE(ad), NULL, 0);
+
+		if (ret) {
+			LOG_ERR("Advertising failed to start (ret %d)", ret);
+			return;
+		}
+
+		LOG_INF("Regular advertising started");
+	}
+}
+
+#if CONFIG_BT_BONDABLE
+static void bond_find(const struct bt_bond_info *info, void *user_data)
+{
+	int ret;
+
+	/* Filter already connected peers. */
+	if (default_conn) {
```

Comment:
Curious: would the code reach here (bond_find during connection)

Reply:
If we want to have more than one connection it might